ERROR:Caught exception: !!!
Traceback (most recent call last):
  File "D:\Lab_python\decorator.py", line 32, in wrapper
    return func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^
  File "D:\Lab_python\models\inkjet_printer.py", line 79, in get_remaining_pages_count
    raise OutOfPaperException ("!!!")
exception.OutOfPaperException: !!!
ERROR:Caught exception: Low paper count!
Traceback (most recent call last):
  File "D:\Lab_python\decorator.py", line 32, in wrapper
    return func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^
  File "D:\Lab_python\models\three_d_printer.py", line 52, in get_remaining_pages_count
    raise OutOfPaperException("Low paper count!")
exception.OutOfPaperException: Low paper count!
